autofit/aggregator/aggregator.py

```diff
@@ -115,6 +115,10 @@ def __init__(
         search_outputs
             A list of search_outputs
         """
+        if len(search_outputs) > 20:
+            print(
+                "When aggregating many searches it can be more efficient to use the database."

```

Comment:
Add following text to message:

"Checkout the database cookbook at this URL: https://pyautofit.readthedocs.io/en/latest/cookbooks/database.html"